moveit_runtime/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package>
+  <name>moveit_runtime</name>
+  <version>0.6.2</version>
+  <description>moveit_runtime meta package contains MoveIt! packages that are essential for its runtime (e.g. running MoveIt! on robots).</description>
+
+  <author email="gm130s@gmail.com">Isaac I. Y. Saito</author>
+  <maintainer email="gm130s@gmail.com">Isaac I. Y. Saito</maintainer>
+
+  <license>BSD</license>
+  <url type="website">http://moveit.ros.org</url>
+  <url type="website">http://wiki.ros.org/moveit_runtime</url>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <run_depend>moveit_commander</run_depend>
+  <run_depend>moveit_core</run_depend>
+  <run_depend>moveit_planners</run_depend>
+  <run_depend>moveit_plugins</run_depend>
+  <run_depend>moveit_ros</run_depend>
```

Comment:
moveit_ros should be separated into its child packages, and moveit_ros_visualization and moveit_benchmarks should not be included